docs/templates.md

```diff
@@ -0,0 +1,24 @@
+# Distributed Application Templates
```

Comment:
Please update the title to Decentralized, Distributed is different from decentralized, it is possible that distributed system is centralized